scripts/builders/palettes/json-to-clr.swift

```diff
@@ -0,0 +1,57 @@
+import AppKit
```

Comment:
This only works on Apple Swift right? Meaning that we'd need to do some extra adjustments in the CI for this to pass?

Reply:
That's correct, I hadn't thought of that.

Reply:
Probably. Should we just change the runner to macos-latest and add the setup swift action in there?

Reply:
Done.